AutoMad/wordGrabber.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Oct  3 11:21:19 2024
+
+@author: dragon
+"""
+
+import os.path
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+import numpy as np
+import pandas as pd
+import madDictionary
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly","https://www.googleapis.com/auth/documents"]
```

Comment:
You could consider including the `SCOPES` in the `token.json` you write out so that the code could check if they have changed and invalidate automatically. But given that you are not likely to change this that may be overkill.